src/components/TaskTracker/InfoModal.tsx

```diff
@@ -30,12 +30,12 @@ export const TaskInfoModal = ({ isVisible = false, onClose }) => {
         </div>
         <div className="border-gray-100 pb-2">
           <div className="rounded pb-2 text-center font-bold">
-            Your tasks tracker
+            Your Task Tracker
           </div>
           <hr className="border-t-3 w-1/4 border-[#5c5c5c]" />
           <div className="items-center p-2 px-2">
-            Double click a task you want to track with the pomodoro counters
-            (will turn blue)
+            Right Click any task to Track, Complete, or Delete any task. 
+            <br></br>
```

Comment:
nit: Can we find another solution instead of having a break point

Reply:
We don't actually need that, it has been removed.

Resolved in [51065bd](https://github.com/MelkeyOSS/astrostation/pull/242/commits/51065bd552e5c9511609ddfc22ff437964cd54bd)